velox/functions/lib/aggregates/AverageAggregateBase.h

```diff
@@ -0,0 +1,412 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/exec/Aggregate.h"
+#include "velox/expression/FunctionSignature.h"
+#include "velox/functions/prestosql/aggregates/AggregateNames.h"
```

Comment:
These includes form a cycle: lib depends on prestosql which depends on lib. Could you please remove includes for prestosql files from lib?

Reply:
Thanks for catching this. Removed this one.
To remove below one, DecimalAggregate.h was moved from prestosql to lib.
```
#include "velox/functions/prestosql/aggregates/DecimalAggregate.h"
```